doc/tutorials/gaussian_transformation.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p>Critical point -&gt; some physicists have a very deep definition of that term. Better "special point" (unless there is a phase transition and singularity)</p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Yes . However, we are talking in a clear mathematical context.</p><p><br></p><p>I will wait for others to pitch in on this. </p>